docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   client:
-    image: "election-verification-site:latest"
+    # image: "election-verification-site:latest"
```

Comment:
Hey @stefan-av , was any of the changes in this file supposed to be pushed? Should I revert them?